var/spack/repos/builtin/packages/blast2go/package.py

```diff
@@ -14,6 +14,11 @@ class Blast2go(Package):
 
     version('5.2.5', sha256='c37aeda25f96ac0553b52da6b5af3167d50671ddbfb3b39bcb11afe5d0643891')
 
+    for t in ['aarch64', 'arm', 'ppc', 'ppc64', 'ppc64le',
+              'ppcle', 'sparc', 'sparc64', 'x86']:
+        conflicts('target={0}:'.format(t),
+                  msg='blast2go is available x86_64 only')
```

Comment:
Can you use:
```python
set([str(x.family) for x in archspec.cpu.TARGETS.values() if str(x.family) != 'x86_64'])
```
instead of an fixed list that may change over time?

Reply:
Thanks for the advice.
I have corrected the points you suggested.